src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/SingleSpaceSeparatorCheck.java

```diff
@@ -77,6 +85,17 @@
  *   &lt;property name=&quot;validateComments&quot; value=&quot;true&quot;/&gt;
  * &lt;/module&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * int foo()   { &#47;&#47;Violation, 3 whitespaces
+ * return 1+2  ; } &#47;&#47;Violation, 2 whitespaces
+ * void fun1() {    &#47;&#47;Violation, 4 whitespaces
+ *   return ; } &#47;&#47;OK
+ * void  fun2() {} &#47;&#47;Violation, 2 whitespaces
+ * void fun3() {  &#47;*Violation here, 2 whitespaces
+ * OK here*&#47; return; }
+ * &#47;*Violation, 2 white spaces*&#47;  int a;
```

Comment:
I see no examples of comments validation, as it stated in config

Reply:
there are 2 white spaces after the comment and before the "int a;" to demonstrate whitespaces surrounding comments will result in violation.